app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivity.kt

```diff
@@ -8,6 +8,12 @@ import org.oppia.android.app.activity.InjectableAppCompatActivity
 import org.oppia.android.app.drawer.KEY_NAVIGATION_PROFILE_ID
 import javax.inject.Inject
 
+const val MARK_CHAPTERS_COMPLETED_FRAGMENT = "MARK_CHAPTERS_COMPLETED_FRAGMENT"
+const val MARK_STORIES_COMPLETED_FRAGMENT = "MARK_STORIES_COMPLETED_FRAGMENT"
+const val MARK_TOPICS_COMPLETED_FRAGMENT = "MARK_TOPICS_COMPLETED_FRAGMENT"
+const val EVENT_LOGS_FRAGMENT = "EVENT_LOGS_FRAGMENT"
+const val FORCE_NETWORK_TYPE_FRAGMENT = "FORCE_NETWORK_TYPE_FRAGMENT"
```

Comment:
Name these as per convention mentioned in this issue: #1587

Reply:
Also possibly mention in alphabetical order.

Reply:
I was refactoring all the PRs and it came to my notice that these are unnecessary. I have removed them now.